docs/why.qmd

```diff
@@ -2,24 +2,224 @@
 title: "Why Ibis?"
 ---
 
+Ibis defines a Python dataframe API that supports any query engine -- the
+frontend for any backend data platform. This includes 20+ backends today, most
+SQL-based.  This allows Ibis to have excellent performance -- as good as the
+backend it is connected to -- with a consistent user experience.
+
+## What is Ibis?
+
 Ibis is the portable Python dataframe library.
 
-If you've had issues with scaling data transformation code in Python, need to
-work with data in multiple data platforms, find yourself translating between
-other Python dataframe APIs, or just want a great Python dataframe experience,
-Ibis is for you.
+We can demonstrate this with a simple example on a few local query engines:
```

Comment:
```suggestion
We can demonstrate this by running the same code against different backends and showing we get the same result.
```

Reply:
I think we can let them see that for themselves :)